src/compiler/emitter.ts

```diff
@@ -6757,13 +6785,32 @@ var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, Promi
                 write("});");
             }
 
+            function lookupSpecifierName(declaration: ImportEqualsDeclaration | ImportDeclaration | ExportDeclaration): string {
+                let specifier: Node;

```

Comment:
`getTargetOfAliasDeclaration` operates over import/export clauses, rather than the parent declarations, so it's very messy and unneeded here.